src/main/java/org/sonar/plugins/github/InputFileCache.java

```diff
@@ -44,6 +44,7 @@ public InputFileCache(GitHubPluginConfiguration gitHubPluginConfiguration, FileS
 
   @Override
   public boolean shouldExecuteOnProject(Project project) {
+    System.out.println("TODO Remove me");
```

Comment:
![New](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/new.png) ![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)